source/opt/set_spec_constant_default_value_pass.cpp

```diff
@@ -248,5 +250,85 @@ bool SetSpecConstantDefaultValuePass::Process(ir::Module* module) {
   return modified;
 }
 
+// A helper function for parsing spec constant default value string. Returns
+// true if the char pointed by the given pointer is considered as the end of
+// the string, which means it is a '\0' or '}'.
+inline bool IsEnd(const char* const ptr) {
+  if (!ptr) return false;
+  return *ptr == '\0' || *ptr == '}';
+}
+
+// A helper function for parsing spec constant default value string. Returns
```

Comment:
Just say what it computes, not what context it's executed in (since that's irrelevant and likely to go stale.)